### ISSUE TITLE
Improve link styles (a11y)

### ISSUE THREAD

**Issue by aviannamiller** [omeka/theme-emiglio#21]
Link to body text contrast should be at least 3:1, it's currently 2.9:1 **on hover**. Otherwise, all set here.
